www/assets/js/components/SearchPage.tsx

```diff
@@ -327,19 +327,23 @@ export default function SearchPage(props: SearchPageProps) {
                   <button
                     onClick={() => updateUI(null)}
                     className="layout-button-left"
+                    aria-label="search results with thumbnails"
```

Comment:
Maybe "Show result thumbnails"

Reply:
I went with "show detailed results" to complement "show compact results".